sources/views/contacts.js

```diff
@@ -16,34 +17,64 @@ export default class Contact extends JetView {
 				},
 
 				deleteIcon: "<span class='remove-btn webix_icon wxi-close'></span>"
+			},
+			onClick: {
+				"wxi-close": (e, id) => this.deleteContact(id)
 			}
 		};
 
-		const userForm = {
-			view: "form",
-			localId: "userForm",
-			elements: [
-				{
-					view: "text",
-					label: "User Name:",
-					name: "name"
-				},
-				{
-					view: "text",
-					label: "Email:",
-					name: "Email"
-				},
-				{}
-			]
+		const addButton = {
+			view: "button",
+			value: "Add",
+			click: () => this.addContact()
 		};
 
 		const view = {
 			margin: 20,
-			cols: [usersList, userForm]
+			cols: [{ rows: [usersList, addButton] }, userForm]
 		};
 		return view;
 	}
-	init() {
-		this.$$("userList").parse(contacts);
+	init(view, url) {
+		this.list = this.$$("userList");
+		this.list.sync(contacts);
+		this.list.attachEvent("onAfterSelect", chosenId => {
+			const previousId = this.getParam("id");
+			const item = contacts.getItem(previousId);
+
+			if (item) {
+				if (!this.formValidation(item)) {
```

Comment:
why do we need to delete any contact after selection? the contact must be deleted only by pressing the cross icon in the list.